backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -120,6 +125,20 @@ private void setPreset(IPCamera camera, List<String> tagList)
     int presetID = presetController.addPreset(createPreset(camera, tagList));
     createImage(camera.getId(), presetID);
   }
+  
+  /**
+   * Creates a preset from a camera.
+   * @param camera    The camera to create the preset from.
+   * @param tagList   The tag belonging to the preset. 
+   * @return          A Preset object.
+   * @throws CameraConnectionException If the camera cannot be reached.
+   */
+  private Preset createSimplePreset(SimpleCamera camera, List<String> tagList) 
+      throws CameraConnectionException {
+    int cameraId = camera.getId();
+    return new Preset(new Position(-1, -1), -1, -1, -1, false, -1,
```

Comment:
Maybe we should create a superclass preset with subclasses SimpleCameraPreset and IPCameraPreset.